src/common/java/net/minecraftforge/gradle/common/util/RunConfig.java

```diff
@@ -85,6 +88,18 @@ public void setArgs(List<String> values) {
         return args;
     }
 
+    public void jvmArg(String value) {
+        getJvmArgs().add(value);
+    }
+    public void setJvmArgs(List<String> values) {
```

Comment:
Can you rename this to `jvmArgs` to prevent confusions on whether this clears previous args?